lib/ansible/module_utils/jboss.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Ansible by Red Hat, inc
```

Comment:
I assume this line is here (and in other files) as you've copied existing modules.